### PR TITLE
Updated README - see description

### PR DESCRIPTION
**Pull request by hwilliams10**
Includes updated usage scenarios and function descriptions.
Also includes example outputs

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,14 +4,30 @@
...
-### Function Descriptions
+```
+pip install -i https://test.pypi.org/simple/ samplingsimulatorpy
+```
+
+
+## Function Descriptions
 
 - `generate_virtual_pop` creates a virtual population.
     - **Inputs** : distribution function (i.e. `np.random.lognormal`, `np.random.binomial`, etc), the paramaters required by the distribution function, and the size of the population.
```
Comment (lisebraaten) [inline README.md]:
```suggestion
    - **Inputs** : distribution function (i.e. `np.random.lognormal`, `np.random.binomial`, etc), the parameters required by the distribution function, and the size of the population.
```

File: README.md [hunk 2]
```diff
@@ -39,23 +55,128 @@ To the best of our knowledge, there is currently no existing Python package with
...
+
+  - `size`: The number of samples
+  - `distribution_func`: The distribution that we are generating samples from
+  - `*para`: The arguments required for the distribution function
+
+**Example:**
+
+`pop <- generate_virtual_pop(100, "height", rnorm, 0, 1)`
+
+#### `draw_samples`
```
Comment (lisebraaten) [inline README.md]: seems to be R syntax
```suggestion
`pop = generate_virtual_pop(100, np.random.normal, 0, 1)`

#### `draw_samples`
```

File: README.md [hunk 3]
```diff
@@ -39,23 +55,128 @@ To the best of our knowledge, there is currently no existing Python package with
...
 
-### Features
-- Will be updated once functions are created
+**Arguments:**
 
-### Dependencies
+  - `pop` the virtual population as a tibble
+  - `reps` the number of replication for each sample size as an integer
+    value
+  - `n_s` the sample size for each one of the samples as an array
```
Comment (lisebraaten) [inline README.md]:
```suggestion
  - `pop` the virtual population as a data frame
  - `reps` the number of replication for each sample size as an integer
    value
  - `n_s` the sample size for each one of the samples as a list
```

File: README.md [hunk 4]
```diff
@@ -39,23 +55,128 @@ To the best of our knowledge, there is currently no existing Python package with
...
+
+**Example:**
 
-- Will be updated once functions are created
+`samples <- draw_samples(pop, 3, [5, 10, 15, 20])`
```
Comment (lisebraaten) [inline README.md]:
```suggestion
`samples = draw_samples(pop, 3, [5, 10, 15, 20])`
```

File: README.md [hunk 5]
```diff
@@ -39,23 +55,128 @@ To the best of our knowledge, there is currently no existing Python package with
...
+``` 
+from samplingsimulatorpy import plot_sample_hist
+plot_sample_hist(pop, samples)
+```
 
-### Usage
+**Arguments:**
+
+  - `pop` the virtual population as a tibble
+  - `samples` the samples as a tibble
```
Comment (lisebraaten) [inline README.md]:
```suggestion
  - `pop` the virtual population as a data frame
  - `samples` the samples as a data frame
```

File: README.md [hunk 6]
```diff
@@ -39,23 +55,128 @@ To the best of our knowledge, there is currently no existing Python package with
...
+#### `plot_sampling_hist`
+
+``` 
+from samplingsimulatorpy import plot_sampling_hist
+plot_sampling_hist(pop, samples)
+```
+
+**Arguments:**
+
+  - `samples` the samples as a tibble
```
Comment (lisebraaten) [inline README.md]:
```suggestion
  - `samples` the samples as a data frame
```

**Review by lisebraaten — CHANGES_REQUESTED** [review]

**Comment by hwilliams10** [comment]
Thanks for catching all those R-isms, Lise!

**Review by lisebraaten — APPROVED** [review]

**Pull request merged**